packages/firebase_auth/firebase_auth/ios/Classes/FLTFirebaseAuthPlugin.m

```diff
@@ -1177,7 +1177,16 @@ - (void)sendSignInLinkToEmailApp:(nonnull AuthPigeonFirebaseApp *)app
            actionCodeSettings:[PigeonParser parseActionCodeSettings:actionCodeSettings]
                    completion:^(NSError *_Nullable error) {
                      if (error != nil) {
-                       completion([FLTFirebaseAuthPlugin convertToFlutterError:error]);
+                       if (error.code == FIRAuthErrorCodeInternalError) {
+                         [self
+                             handleInternalError:^(PigeonUserCredential *_Nullable creds,
+                                                   FlutterError *_Nullable error) {
```

Comment:
Can `error` be renamed? Just so it is a bit clearer it is different from the `error` in the outer scope.

Reply:
renamed